doc/hotspot-style.md

```diff
@@ -747,6 +747,11 @@ part of the avoidance of the C++ Standard Library in HotSpot code.
 ([n3206](http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2010/n3206.htm)),
 ([n3272](http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2011/n3272.htm))
 
+* `override` virtual specifiers for virtual functions
+([n2928](http://www.open-std.org/JTC1/SC22/WG21/docs/papers/2009/n2928.htm)),
+([n3206](http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2010/n3206.htm)),
+([n3272](http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2011/n3272.htm))
```

Comment:
nit - The difference in case caught my attention:

    JTC1 <-> jtc1
    SC22 <-> sc22
    WG21 <-> wg21

any particular significance to the difference?

Reply:
That's how they were written in the document I cut-and-pasted them from (https://gcc.gnu.org/projects/cxx-status.html).  Some of the links there seem to be to systems with case sensitive file-systems.